BTMLColorLOSMod/WeaponRangeIndicators_DrawLine_Patch.cs

```diff
@@ -189,16 +201,25 @@ public static class WeaponRangeIndicators_DrawLine_Patch
                         {
                             // other than formatting this block is the only thing that changed from the decompiled code
                             Vector3[] pointsForArc = WeaponRangeIndicators.GetPointsForArc(18, 30f, vector, vector2);
+
                             float shotQuality = (float) ReflectionHelper.InvokePrivateMethode(__instance,
                                 "GetShotQuality", new object[] {selectedActor, position, rotation, target});
                             // alright future me, this is probably destructive in some way, but 
                             // this lets us set the color of the line via that color6 bit.
-                            line.material.color = Color.white;
                             Color color6 = Color.Lerp(
                                 Color.clear,
-                                BTMLColorLOSMod.ModSettings.IndirectLineOfFireArcColor, 
+                                BTMLColorLOSMod.ModSettings.IndirectLineOfFireArcColor,
                                 shotQuality);
-                            line.endColor = (line.startColor = color6);
+                            if (BTMLColorLOSMod.ModSettings.IndirectLineOfFireArcDashed)
+                            {
+                                line.material = __instance.MaterialOutOfRange;
```

Comment:
So the default line width is too small to make this readable. I popped in:
```
line.endWidth = __instance.LOSWidthEnd * 1.75f;      // dashes are harder to see,
line.startWidth = __instance.LOSWidthBegin * 1.75f;  // so make 'em bigger
```

Original width:
<img width="420" alt="screen shot 2018-05-21 at 12 28 51 am" src="https://user-images.githubusercontent.com/50124/40290964-c6f5a842-5c8f-11e8-9453-5abb1629d9a7.png">

Bigger width:
<img width="478" alt="screen shot 2018-05-21 at 12 40 23 am" src="https://user-images.githubusercontent.com/50124/40290965-c7101998-5c8f-11e8-95f2-68116930e241.png">